modules/common/istio/istio-values.tpl

```diff
@@ -40,10 +40,10 @@ global:
           memory: 10Mi
     resources:
       requests:
-        cpu: 100m
+        cpu: 1000m
         memory: 128Mi
       limits:
-        cpu: 1000m
+        cpu: 3000m
         memory: 512Mi
```

Comment:
I think these are the default resources for the sidecar istio creates (not for the pilot container) but I could be wrong. Did you test this in sandbox to see what resources it changes?